src/views/Restoration/RestorationBusinessName.vue

```diff
@@ -22,7 +22,7 @@
       </header>
       <v-card flat class="mt-5">
         <RestorationType id="restoration-type" />
-        <ApprovalType id="approval-type" class="mt-n8" />
+        <ApprovalType id="approval-type" class="mt-n10" />
```

Comment:
This can be fixed later, but it looks like ApprovalType has too much bottom whitespace in it. Again, components should have "tight" spacing (no whitespace around them), and then the parent can add however much whitespace it needs.

Reply:
The problem is actually the error message of the number of month(s) text field. It's taking a lot of space and pushing things downward. Another fix would be to decrease the height of it in the shared component.

Reply:
There may be a (Vuetify) component property to hide the error message when it's empty, thus removing that abundant bottom whitespace. However, that may cause screen jerkiness when the message is shown/hidden -- need to test and see which is best.

Yes, the shared component should not have excess whitespace. But that can be fixed later.